src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -1959,7 +2008,7 @@ public void RestoreFirstClearsProjectRootElementCache()
                 }
             };
 
-            string logContents = ExecuteMSBuildExeExpectSuccess(projectContents, preExistingProps, "/restore");
+            string logContents = ExecuteMSBuildExeExpectSuccess(projectContents, filesToCreate: preExistingProps, arguments: "/restore");
```

Comment:
nit:
I don't like this sort of change, but you can wait for others' opinions if you'd like.